honeybee_radiance/cli/grid.py

```diff
@@ -0,0 +1,78 @@
+"""honeybee radiance grid commands.""" 
+
+try:
+    import click
+except ImportError:
+    raise ImportError(
+        'click is not installed. Try `pip install . [cli]` command.'
+    )
+
+import sys
+import honeybee_radiance.sensorgrid as sensorgrid
+import os
+import logging
+import re
+
+_logger = logging.getLogger(__name__)
+
+@click.group(help='Commands for generating and modifying sensor grids.')
+def grid():
+    pass
+
+@grid.command('split')
+@click.argument('grid-file')
+@click.argument('count', type=int)
+@click.option('--folder', help='Optional output folder.', default='.', show_default=True)
+def split_grid(grid_file, count, folder):
+    """Split a radiance grid file into smaller grids based on maximum sensor count.
+
+    \b
+    Args:
+        grid-file: Full path to input sensor grid file.
+        count: Maximum number of sensors in new files. The number will be rounded to
+            closest round number for each file. For example if the input file has 21
+            sensors and input count is set to 5 this command will generate 4 files where
+            the first three files will have 5 sensors and the last file will have 6. 
+    """
+    try:
+        grid = sensorgrid.SensorGrid.from_file(grid_file)
+        file_count = int(round(grid.count / count))
+        files = grid.to_files(folder, file_count, mkdir=True)
+        for f in files:
+            click.echo(f)
+    except Exception:
+        _logger.exception('Failed to split grid file.')
+        sys.exit(1)
+
+
+@grid.command('merge')
+@click.argument('input-folder')
+@click.argument('base-name')
+@click.argument('extension', default='.pts')
+@click.option('--folder', help='Optional output folder.', default='.', show_default=True)
+def merge_grid(input_folder, base_name, extension, folder):
+    """Merge several radiance grid files into a single file.
+
+    \b
+    Args:
+        input_folder: Input folder.
+        base_name: File base name. All the file must star with base name and continue
+            with _ and integer values.
+        extension: File extention. [Default: .pts]
+    """
+    try:
+        pattern = r'{}_\d+{}'.format(base_name, extension)
+        grids = sorted(f for f in os.listdir(input_folder) if re.match(pattern, f))
```

Comment:
Just a thought here, when we merge point files together using this method, is there a way for us to ensure that we can then match each point line back to a simulation point output file line?

Reply:
With files being sorted by names then each line in grid file will correspond to the same line in result file. I actually also use this method to merge the results back together. That's why there is an input for extension. In most cases we already have the original pts file.